watertap/core/wt_database.py

```diff
@@ -0,0 +1,111 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+This module contains the base class for interacting with WaterTAP data files
+with zero-order model parameter data.
+"""
+import os
+import yaml
+
+
+class Database:
+    def __init__(self, dbpath=None):
+        self._cached_files = {}
+
+        if dbpath is None:
+            self._dbpath = os.path.join("watertap", "data", "techno_economic")
```

Comment:
```suggestion
            self._dbpath = os.path.join(os.path.dirname(os.path.abspath(__file__)),"..", "data", "techno_economic")
```
I think something like this needs to be the default, so the default `_dbpath` will get set correctly no matter where `watertap` happens to be installed. Note that `os.path.dirname(os.path.abspath(__file__))` should return `watertap/core`.